src/test/java/co/rsk/federate/FederationWatcherTest.java

```diff
@@ -31,26 +32,23 @@
 import org.mockito.invocation.InvocationOnMock;
 
 class FederationWatcherTest {
-    private final Federation federation1 = FederationFactory.buildStandardMultiSigFederation(
-            getFederationMembersFromPksForBtc(1000, 2000, 3000, 4000),
-            Instant.ofEpochMilli(5005L),
-            0L,
-            NetworkParameters.fromID(NetworkParameters.ID_REGTEST)
-    );
-
-    private final Federation federation2 = FederationFactory.buildStandardMultiSigFederation(
-            getFederationMembersFromPksForBtc(2000, 3000, 4000, 5000, 6000, 7000),
-            Instant.ofEpochMilli(15300L),
-            0L,
-            NetworkParameters.fromID(NetworkParameters.ID_REGTEST)
-    );
-
-    private final Federation federation3 = FederationFactory.buildStandardMultiSigFederation(
-            getFederationMembersFromPksForBtc(5000, 6000, 7000),
-            Instant.ofEpochMilli(7400L),
-            0L,
-            NetworkParameters.fromID(NetworkParameters.ID_REGTEST)
-    );
+    List<FederationMember> federation1Members = getFederationMembersFromPksForBtc(1000, 2000, 3000, 4000);
+    Instant federation1CreationTime = Instant.ofEpochMilli(5005L);
+    long creationBlockNumber = 0L;
+    NetworkParameters btcParams = NetworkParameters.fromID(NetworkParameters.ID_REGTEST);
+
+    FederationArgs federation1Args = new FederationArgs(federation1Members, federation1CreationTime, creationBlockNumber, btcParams);
+    private final Federation federation1 = FederationFactory.buildStandardMultiSigFederation(federation1Args);
+
+    List<FederationMember> federation2Members = getFederationMembersFromPksForBtc(2000, 3000, 4000, 5000, 6000, 7000);
+    Instant federation2CreationTime = Instant.ofEpochMilli(15300L);
```

Comment:
Do these values have a meaning? `15300L` is something important or could it be any other random value? If it needs to be exactly this value, then we could put it in a static final variable. If not, that's fine.